UnitTests/Plugins/BuildServerIntegration/AzureDevOpsIntegration.Tests/AzureDevOpsAdapterTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using AzureDevOpsIntegration;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace AzureDevOpsIntegrationTests
+{
+    [TestFixture]
+    public class AzureDevOpsAdapterTests
+    {
+        [Test]
+        public void Should_not_filter_running_builds_When_only_one_returned()
+        {
+            var adapter = new AzureDevOpsAdapter();
+            var testAccessor = adapter.GetTestAccessor();
```

Comment:
[fyi] These could be moved into `Setup` method, so all tests don't have to do this.